lib/correios/cep/address.ex

```diff
@@ -3,16 +3,16 @@ defmodule Correios.CEP.Address do
   Address structure.
   """
 
-  @enforce_keys [:street, :complement, :neighborhood, :city, :state, :zipcode]
-
-  defstruct @enforce_keys
+  @enforce_keys [:street, :complement, :neighborhood, :city, :state]
+  defstruct @enforce_keys ++ [:postal_code, :zipcode]
```

Comment:
Why not are `postal_code` and `zipcode` in `@enforce_keys`?

Reply:
Maybe I miss understand; If I keep both fields `postal_code` and `zipcode` it will not make both fields required to be filled when using the struct?

Reply:
I thought that I was going to need two "constructors", one that allows using zipcode and another one that allows using postal_code.

To be clear, I need to add both fields in the enforce_keys and keep a single `new` function with these keys in the map, but both will fill the value for `postal_code`, is that correct?

Reply:
The expected response should have both `postal_code` and `zipcode` fields:
```
iex> Correios.CEP.find_address("54250610")
{:ok,
 %Correios.CEP.Address{
   city: "Jaboatão dos Guararapes",
   complement: "",
   neighborhood: "Cavaleiro",
   postal_code: "54250610",
   state: "PE",
   street: "Rua Fernando Amorim",
   zipcode: "54250610"
 }}
```

That way does not break the current consumers and give them the opportunity to start using `postal_code` instead of the current `zipcode` that would be removed in a future version.